docs/solutions/build-errors/validate-agent-authoring-changelog-skip.md

```
@@ -0,0 +1,104 @@
+  - tests/integration/validate-agent-authoring-changelog-skip.test.ts
+related-prs:
+  - audit/x-02-validator-frontmatter-only
+---
+
+# `validate-agent-authoring.js` CHANGELOG.md skip
+
+## Problem
+
+`pnpm release:check` fails with a hard ERROR like:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)